lms/envs/common.py

```diff
@@ -1860,6 +1860,9 @@
     # Microsite configuration
     'microsite_configuration',
 
+    # RSS Proxy
+    'rss_proxy',
```

Comment:
I think the jury is going to be out on this approach forever -- in general we do not want to continue adding to the sprawl of apps that make up the core LMS, we want nicely-packaged components which can be updated individually and used (or not used) by other installations and services.  We don't have to hold up this particular PR, but it would have been nice to see more alignment with the general strategy, even if not entirely baked at this point.